var/spack/repos/builtin/packages/py-fparser/package.py

```diff
@@ -10,18 +10,20 @@ class PyFparser(PythonPackage):
     """Parser for Fortran 77..2003 code."""
 
     homepage = "https://github.com/stfc/fparser"
-    url      = "https://github.com/stfc/fparser/archive/0.0.5.tar.gz"
-    git      = "https://github.com/stfc/fparser.git"
+    # url      = "https://github.com/stfc/fparser/archive/0.0.5.tar.gz"
+    # git      = "https://github.com/stfc/fparser.git"
+    pypi     = "fparser/fparser-0.0.16.tar.gz"
 
-    version('develop', branch='master')
-    version('0.0.6', sha256='6ced61573257d11037d25c02d5f0ea92ca9bf1783018bf5f0de30d245ae631ac')
-    version('0.0.5', sha256='7668b331b9423d15353d502ab26d1d561acd5247882dab672f1e45565dabaf08')
+    # version('develop', branch='master')
+    version('0.0.16', sha256='a06389b95a1b9ed12f8141b69c67343da5ba0a29277b2997b02573a93af14e13')
+    version('0.0.6', sha256='bf8a419cb528df1bfc24ddd26d63f2ebea6f1e103f1a259d8d3a6c9b1cd53012')
+    version('0.0.5', sha256='f3b5b0ac56fd22abed558c0fb0ba4f28edb8de7ef24cfda8ca8996562215822f')
 
     depends_on('py-setuptools', type='build')
 
     depends_on('py-numpy', type=('build', 'run'), when='@:0.0.5')
-    depends_on('py-nose', type='build')
-    depends_on('py-six', type='build', when='@0.0.6:')
+    depends_on('py-nose', type=('build', 'run'), when='@:0.0.7')
+    depends_on('py-six', type=('build', 'run'), when='@0.0.6:0.0.14')
```

Comment:
Six is still required in the latest release

Reply:
I did some digging and saw that it was removed in release 0.0.15, but it was still listed in `setup.py`despite not being used. I

Reply:
If it's listed in the `setup.py` I tend to add it even if it isn't used. You can bug the developers to correct the list of required deps though.